nix/nixos/cardano-db-sync-service.nix

```diff
@@ -177,6 +185,20 @@ in {
             (if (cfg.environment.nodeConfig.RequiresNetworkMagic == "RequiresNoMagic" )
               then "--mainnet"
               else "--testnet-magic $(jq '.networkMagic' ${cfg.environment.nodeConfig.ShelleyGenesisFile})");
```

Comment:
Just a thought, could we get the networkMagic in nix rather than going through `jq`?